### PR TITLE
Add Intel MKL backend

### PR DESCRIPTION
**Pull request by termoshtt**
Same as https://github.com/blas-lapack-rs/lapack-src/pull/1

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ keywords = ["linear-algebra"]
 accelerate = ["accelerate-src"]
 netlib = ["netlib-src"]
 openblas = ["openblas-src"]
+intel-mkl = ["intel-mkl-src"]
```
Comment (IvanUkhov) [inline Cargo.toml]: Let’s keep it sorted everywhere. Please put `intel-mkl` after `accelerate`.

File: README.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ The following implementations are available:
 * `accelerate`, which is the one in the [Accelerate] framework (macOS only),
 * `netlib`, which is the reference one by [Netlib], and
 * `openblas`, which is the one in [OpenBLAS].
+* `intel-mkl`, which is the one in [Intel MKL].
```
Comment (IvanUkhov) [inline README.md]: When you move this one, it should end with a comma.

File: src/lib.rs [hunk 3]
```diff
@@ -9,6 +9,7 @@
 //! * `accelerate`, which is the one in the [Accelerate] framework (macOS only),
 //! * `netlib`, which is the reference one by [Netlib], and
 //! * `openblas`, which is the one in [OpenBLAS].
+//! * `intel-mkl`, which is the one in [Intel MKL].
```
Comment (IvanUkhov) [inline src/lib.rs]: The same here about punctuation.

File: Cargo.toml (hunk 1 above)
Comment (termoshtt) [reply Cargo.toml]: - accelerate
- intel-mkl
- openblas
- netlib

right?

Comment (IvanUkhov) [reply Cargo.toml]: Yes, great!

**Comment by termoshtt** [comment]
Sorry for dirty commits...

**Comment by termoshtt** [comment]
I forget to add CI test

**Comment by IvanUkhov** [comment]
Thank you!

**Comment by termoshtt** [comment]
macOS instance of Travis seems to go bad https://www.traviscistatus.com/
Would you please upload the 0.1.3 to crate.io?

**Comment by IvanUkhov** [comment]
Done!

**Comment by termoshtt** [comment]
Thanks!

**Comment by IvanUkhov** [comment]
@termoshtt, you might also want to update [this page](https://blas-lapack-rs.github.io/usage).

**Pull request merged**